PIMS/Controllers/ProductsController.cs

```diff
@@ -0,0 +1,152 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Services.Helper;
+using ServicesContract.Model;
+using ServicesContracts;
+using ServicesContracts.Model;
+using System;
+using System.Threading.Tasks;
+
+namespace Assignment_1.Controllers
+{
+    [Route("/products")]
```

Comment:
Simplified route to remove the leading slash

Reply:
Resolved